src/config/query/query_config.py

```diff
@@ -0,0 +1,80 @@
+"""
+    Module for loading queries from yaml.
+"""
+import yaml
+
+FILE_PATH = 'src\\config\\query\\query_collector.yaml'
+
+class QueryConfig:
+    """
+        Class for loading queries from yaml.
+    """
+    @classmethod
+    def load(cls):
+        with open(FILE_PATH, 'r') as file:
+            data = yaml.safe_load(file)
+            
+            # table headers
+            cls.employee_detail_header = data["employee_detail_header"]
+            cls.customer_detail_header = data["customer_detail_header"]
```

Comment:
Instead of creating a yml you can simply create .py file and and create all these constant as class variables.
that way you wont have to read from file.